src/utils.js

```diff
@@ -0,0 +1,18 @@
+export function sortByTags(tags = [], source = [{}]) {
+  let projects = source
```

Comment:
Do you need to use "let" to define this variable or could you also use "const"?

Reply:
`const` is also possible, thanks